### ISSUE TITLE
Royal Signet Ring in TES3MP?

### ISSUE THREAD

**Issue by Javimel** [TES3MP/TES3MP#487]
TL;DR Is there a way to allow players to equip the RSR? The chatbox in multiplayer says "you're not allowed to equip that".

I'm incredibly new to Morrowind as a whole, played maybe 10 minutes of the original on xbox, then decided to pick it up with a friend to play the amazing multiplayer that had been integrated to do some shenanigans. I'm Roguey, Thiefy, I like taking things when people aren't looking, or even if they are.

Imagine my surprise when I steal the Royal signet ring (and everything else) from King Hlaalu, with the "take all" option, in addition to his unique robes, and the dagger that's normally a quest reward. Needless to say, I was hyped.

Only to find out that the game itself won't allow me to equip the ring. Can anyone help me? Is there a setting in a config file that I don't know about? Is there currently any way to enable the use of the best ring in the game?
